presto-thrift-connector/src/main/java/com/facebook/presto/connector/thrift/clientproviders/RetryingPrestoThriftServiceProvider.java

```diff
@@ -178,4 +185,56 @@ public void close()
             client = null;
         }
     }
+
+    public static class RetryingPrestoThriftServiceStats
```

Comment:
This can be named just `Stats` as it is a nested class

Reply:
Changing it